modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtInstancelevel.testcase

```diff
@@ -0,0 +1,329 @@
+
+		task ("Then the list of fields to select contains all the fields available with GET request except the relationship nodes") {
+			var fieldList1 = "name,accountStudents,accountUserAccounts,actions,customFields,dateCreated,dateModified";
+			var fieldList2 = "defaultBillingAddressId,defaultShippingAddressId,description,domains,externalReferenceCode,id,logoId";
+			var fieldList3 = "logoURL,numberOfUsers,organizationIds,parentAccountId,postalAddresses,status,taxId,type";
+
+			var exportFields = "${fieldList1},${fieldList2},${fieldList3}";
+
+			ImportExport.mapExport(exportFields = ${exportFields});
+		}
+	}
+
+	@priority = 5
+	test CanDownloadFiledWithAllFieldsSelected {
+		task ("And Given 'C_Student (v1_0 - Liferay Object REST)' as Entity Type in Data Migration Center --> Export File") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.gotoExport();
+
+			ImportExport.selectEntity(entityType = "C_Student (v1_0 - Liferay Object REST)");
+		}
+
+		task ("When exporting file in JSON format with all fields selected") {
+			Button.click(button = "Export");
+
+			WaitForElementPresent(locator1 = "ImportExport#EXECUTION_SUCCESS");
+		}
+
+		task ("Then all fields values available in the downloaded file") {
+			ImportExport.unzipDownloadedExportFile();
+
+			var filePath = TestCase.getDependenciesDirPath(fileName = "json_objectEntry_export_all_fields.txt");
+			var response = ObjectDefinitionAPI.getObjectEntries(en_US_plural_label = "students");
+			var userId = JSONUserAPI._getUserIdByEmailAddress(userEmailAddress = "test@liferay.com");
+			var entryId = JSONPathUtil.getProperty(
+				property = "$..items..[0]id",
+				response = ${response});
+			var dateCreated = JSONPathUtil.getProperty(
+				property = "$.items..dateCreated",
+				response = ${response});
+			var dateModified = JSONPathUtil.getProperty(
+				property = "$.items..dateModified",
+				response = ${response});
+			var externalReferenceCode = JSONPathUtil.getProperty(
+				property = "$.items..externalReferenceCode",
+				response = ${response});
+			var accountEntryId = JSONPathUtil.getProperty(
+				property = "$.items..r_accountStudents_accountEntryId",
+				response = ${response});
+			var accountEntryERC = JSONPathUtil.getProperty(
+				property = "$.items..r_accountStudents_accountEntryERC",
+				response = ${response});
+
+			FileUtil.replaceStringInFile(${filePath}, 20123, ${userId});
+
+			FileUtil.replaceStringInFile(${filePath}, 39652, ${entryId});
+
+			FileUtil.replaceStringInFile(${filePath}, "2023-10-19T09:07:36Z", ${dateCreated});
+
+			FileUtil.replaceStringInFile(${filePath}, "2023-10-19T09:07:36Z", ${dateModified});
+
+			FileUtil.replaceStringInFile(${filePath}, "7f1d7aa8-18ce-750c-db70-3e2af227afd8", ${externalReferenceCode});
+
+			FileUtil.replaceStringInFile(${filePath}, 39649, ${accountEntryId});
+
+			FileUtil.replaceStringInFile(${filePath}, "66ff0575-feb9-38fc-ae9d-a5fec2248b23", ${accountEntryERC});
+
+			ImportExport.assertFileContent(
```

Comment:
1. Can we use response from GET request instead of json_objectEntry_export_all_fields.txt? So that we don't need to replace so many values.
2. If we can't use response from GET request, we'd better to replace the fileds in a for loop to avoid so many codes here

Reply:
https://github.com/liferay-headless/liferay-portal/pull/1606#discussion_r1368229214